docs/user-guide/slicing.ipynb

```diff
@@ -6,10 +6,19 @@
    "source": [
     "# Slicing\n",
     "\n",
-    "Data in a [Variable](../generated/scipp.Variable.rst#scipp.Variable) or [Dataset](../generated/scipp.Dataset.rst#scipp.Dataset) can be indexed in a similar manner to NumPy and xarray.\n",
+    "Objects in scipp can be sliced in two ways. The most common is by by index based slicing. A second, more limited approach is to use value based slicing based on coordinate values."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Index based slicing\n",
+    "\n",
+    "Data in a [Variable](../generated/scipp.Variable.rst#scipp.Variable), [Dataset](../generated/scipp.Dataset.rst#scipp.Dataset) or [DataArray](../generated/scipp.DataArray.rst#scipp.DataArray) can be indexed in a similar manner to NumPy and xarray.\n",
     "The dimension to be sliced is specified using a dimension label and, in contrast to NumPy, positional dimension lookup is not available.\n",
-    "Positional indexing with an integer or an integer range is using `__getitem__` and `__setitem__` with a dimension label as first argument.\n",
-    "This is available for variables, datasets, as well as items of a dataset.\n",
+    "Positional indexing with an integer or an integer range is made via `__getitem__` and `__setitem__` with a dimension label as first argument.\n",
+    "This is available for Variables, DataArrays, Datasets, as well as items of a Dataset.\n",
```

Comment:
The documentation is not using the camel case class names, unless referring to the classes. If instances are meant (or if it does not matter) we are using the normal lower-case spelling.